src/design-guidelines/declutter/examples/tooltip-declutter.tsx

```
@@ -8,52 +8,46 @@ import { Component, h } from '@stencil/core';
-                            <p>
-                                Scheduled tasks will be run at night between
-                                01:00 to 02:00.
-                            </p>
-                            <div class="action-bar">
-                                <limel-button label="Cancel" />
-                                <limel-button label="Schedule" primary={true} />
-                            </div>
+            <limel-example-do-do-not
+                doDescription="When a UI is regularly used to perform a task, provide the information that most users already know, in a tooltip."
+                doNotDescription="Avoid having UIs that are cluttered with information that most users already know."
+            >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)